packages/@okta/vuepress-site/docs/reference/token-hook/index.md

```diff
@@ -377,7 +385,10 @@ Use the `add` operation to add new claims to a token. If you use the `add` opera
         }
       ]
     }
-  ]
+  ],
+"error": {
+    "errorSummary": "User is not a patient."
+}
```

Comment:
I don't think we can have an error response at the same time as it includes the `commands` attribute? I could be wrong but I didn't think so.